_data/locales/it.yml

```diff
@@ -0,0 +1,124 @@
+      automation:
+        title: "Automazione"
+        description: '<a href="https://opensource.guide/best-practices/#bring-in-the-robots">Coinvolgi i robot</a> per automatizzare il più possibile attraverso test e altri controlli per risparmiare tempo.'
+    visitor:
+      title: "Preparatevi per l'Open Source Friday"
+      description: "Trascorri del tempo ora per risparmiare tempo in seguito."
+  endorsement:
+    title: "Sei in buona compagnia"
+    Kim_Crayton:
+      title: "Ingegnere comunitario, scrittore, fondatore"
+      name: "Kim Crayton"
+      message: "Contribuire all'open source non solo consente alle aziende di ogni dimensione di trarre vantaggio e sfruttare i talenti degli altri, ma è anche una strada per coloro che sono nuovi alla programmazione per acclimatarsi al processo di sviluppo in modi che consentano loro di esercitarsi, sperimentare e mostrare stessi come membri preziosi e contribuenti della comunità."
+    Chris_Lamb:
+      title: "Responsabile del <a href='https://www.debian.org'>progetto Debian</a>"
+      name: "Chris Lamb"
+      message: "Sono arrivato dove sono oggi contribuendo al software open source, permettendomi non solo di migliorare le mie capacità di sviluppatore di software ma anche di aumentare la mia rete di colleghi e di incontrare nuovi e interessanti amici provenienti da culture estremamente diverse. Lo consiglio vivamente."
+    Will_Norris:
+      title: "Open Source su <a href='https://github.com/google'>Google</a>"
+      name: "Will Norris"
+      message: 'Contribuire a progetti open source continua a essere uno dei progetti più apprezzati dal 20% degli ingegneri di Google. L'open source svolge un ruolo così importante in gran parte di ciò che facciamo, quindi cerchiamo di rendere <a href="https://opensource.google.com/docs/patching/"> il processo</a> il più semplice possibile possibile.'
+    Christine_Abernathy:
+      title: "Open Source su <a href='https://github.com/facebook/'>Facebook</a>"
+      name: "Christine Abernathy"
+      message: "L'open source è in linea con la nostra missione di rendere il mondo più aperto e connesso. Lavorare con la comunità ci dà l'opportunità di condividere il nostro lavoro e le sfide che miriamo a risolvere e incoraggia le persone di talento a contribuire con le loro idee e i loro sforzi per far progredire il mondo."
+    Rich_Harris:
+      title: "<a href='https://github.com/rollup/rollup'>Rollup</a> e <a href='https://github.com/sveltejs/svelte'>Svelte</a> manutentore"
+      name: "Rich Harris"
+      message: "Ciò che manca alla maggior parte dei progetti open source non è il denaro, ma il tempo. Molti manutentori hanno un lavoro a tempo pieno e non possono realisticamente ridurre le proprie ore. Ciò di cui quasi tutti i progetti hanno bisogno sono più contributori. L'Open Source Friday è un modo perfetto per le aziende di supportare l'ecosistema open source da cui dipendono in modo significativo."
```

Comment:
```suggestion
      message: "Ciò che manca alla maggior parte dei progetti open source non è il denaro, ma il tempo. Molti manutentori hanno un lavoro a tempo pieno e non possono realisticamente ridurre le proprie ore. Ciò di cui quasi tutti i progetti hanno bisogno sono più contributori. L’Open Source Friday è un modo perfetto per le aziende di supportare l’ecosistema open source da cui dipendono in modo significativo."
```

Reply:
I apologize for the delay. I made the requested changes.